merkletree/proof_test.go

```diff
@@ -63,13 +69,7 @@ func verifyProof(t *testing.T, ap *AuthenticationPath, treeHash []byte, key stri
 		}
 	} else {
 		// proof of absence
-		// step 2. vrf_verify(i, alice) == true
-		// we probably want to use vrf.Verify() here instead
-		if !bytes.Equal(vrf.Compute([]byte(key), vrfPrivKey1), ap.LookupIndex()) {
-			t.Error("VRF verify returns false")
-		}
-
-		// step 3. Check that where i and j differ is at bit l
+		// step 3. Check that where i and j match in the first l bits
```

Comment:
Shouldn't this be: "Check if i and j differ in the first l bits" (at least that is what is happening below). 
(Also there is a typo "indecies" here: https://github.com/coniks-sys/coniks-go/pull/38/files#diff-5b678c600e78be51602b1116649c7573L78)